models/category.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+const categorySchema = new mongoose.Schema({
+    categoryName: {
+        type: String,
+        required: true
+    }
+});
+
+const Category = mongoose.model('Category', categorySchema);
```

Comment:
Always pluralize your model name
```suggestion
const Category = mongoose.model('Categories', categorySchema);
```